x/harvest/keeper/borrow_test.go

```diff
@@ -43,11 +45,23 @@ func (suite *KeeperTestSuite) TestBorrow() {
 				contains:   "",
 			},
 		},
+		{
+			"loan-to-value limited",
+			args{
+				borrower:                  sdk.AccAddress(crypto.AddressHash([]byte("test"))),
+				depositCoin:               sdk.NewCoin("ukava", sdk.NewInt(20)),
```

Comment:
update test so we are supplying $100 worth of kava and borrowing $61 dollars of usdx with a max LTV of 60%.

Reply:
Updated in https://github.com/Kava-Labs/kava/pull/710/commits/23dc0a173210adebb04d509ccbae4dd650ede553.

I'll be more careful about matching tests to card specifications going forward.